Docs: Fixup smaller issues (#3610)
cobbler/cobbler: docs/cobbler-conf/settings-yaml.rst

bootloaders_dir
###############

> This is the directory that "cobbler mkloaders" will copy the built bootloaders into. "cobbler sync" will search for

Reviewer: ```suggestion
A directory that "cobbler mkloaders" copies the built bootloaders into. "cobbler sync" searches for
```

grub2_mod_dir
#############

> This is the folder where Cobbler will look for GRUB modules that are required for "cobbler mkloaders".
> 
> default: Depending on your distro. See values below.

Reviewer: I have noticed a lot of `directory` vs `folder` in the Cobbler docs. IMHO using "folder" is a bit weird in a Linux-based system.

Also, you can typically use present tense instead of future tense, which is then shorter and easier to understand for readers.

```suggestion
The directory where Cobbler looks for GRUB modules that are required for "cobbler mkloaders".

default: Depends on your distribution. See values below.
```

* (open)SUSE: ``"/usr/share/grub2"``
* Debian/Ubuntu: ``"/usr/lib/grub"``
* CentOS/Fedora: ``"/usr/lib/grub"``

syslinux_dir
############

> This is the folder where Cobbler will look for syslinux modules that are required for "cobbler mkloaders".
> 
> default: Depending on your distro. See values below.

Reviewer: Same as above

```suggestion
The directory where Cobbler looks for syslinux modules that are required for "cobbler mkloaders".

default: Depends on your distribution. See values below.
```

* (open)SUSE: ``"/usr/share/syslinux"``
* Debian/Ubuntu: ``"/usr/lib/syslinux/modules/bios/"``
* CentOS/Fedora: ``"/usr/share/syslinux"``

bootloaders_modules
###################

> This is a list of all modules that will be included when "cobbler mkloaders" build the grub used for PXE or HTTP Boot.

Reviewer: Side note: You can also safely skip "this is" in docs. 

```suggestion
A list of all modules "cobbler mkloaders" includes when building grub loaders.
Typically, a grub loader uses the modules for PXE or HTTP Boot.
```

grub2_mod_dir
#############

The directory where Cobbler looks for GRUB modules that are required for "cobbler mkloaders".

default: Depends on your distribution. See values below.

* (open)SUSE: ``"/usr/share/grub2"``
* Debian/Ubuntu: ``"/usr/lib/grub"``
* CentOS/Fedora: ``"/usr/lib/grub"``

syslinux_dir
############

The directory where Cobbler looks for syslinux modules that are required for "cobbler mkloaders".

default: Depends on your distribution. See values below.

* (open)SUSE: ``"/usr/share/syslinux"``
* Debian/Ubuntu: ``"/usr/lib/syslinux/modules/bios/"``
* CentOS/Fedora: ``"/usr/share/syslinux"``

bootloaders_modules
###################

A list of all modules "cobbler mkloaders" includes when building grub loaders.
Typically, a grub loader uses the modules for PXE or HTTP Boot.

> default: Omited for readablity, please refer to the `settings.yaml` file in our GitHub repository.

Reviewer 2: Just a nitpick: It would be great if you could link to the settings file you are referring to here, if that's possible. This would make the process for the user a bit quicker.

Author: I think that this is not an easy fix as we would have to adjust the links for all branches manually at the moment. I will open a low-priority docs issue that will mention this after merging this PR. For now I would resolve this conversation to prioritize the performance SUSE-Bugzilla issue that this PR needs to be backported to.
